cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconciler.java

```diff
@@ -281,6 +289,25 @@ public Future<Void> reconcile(KafkaStatus kafkaStatus, Clock clock)    {
                 .compose(i -> addListenersToKafkaStatus(kafkaStatus));
     }
 
+    protected Future<Set<Integer>> brokerScaleDownCheck() {
```

Comment:
Why does this return the `Set<Integer>` and not something like `Void`? You do not use the outcome anywhere-